retrieval/discovery/gce.go

```diff
@@ -177,13 +179,25 @@ func (gd *GCEDiscovery) refresh() (tg *config.TargetGroup, err error) {
 			addr := fmt.Sprintf("%s:%d", priIface.NetworkIP, gd.port)
 			labels[model.AddressLabel] = model.LabelValue(addr)
 
+			// Tags in GCE are mostly used for networking rules (unlike e.g. AWS EC2 tags).
```

Comment:
No coupling in comments, please. If I don't know AWS, this doesn't  help. If AWS or our integration with it changes, it may invalidate comments in the GCE integration code, etc.

Let's just stick with describing whatever necessary explicitly.